airflow/www/static/js/dags.js

```diff
@@ -140,7 +146,21 @@ $(".typeahead").typeahead({
       window.location = `${gridUrl.replace("__DAG_ID__", value.name)}?${query}`;
     }
   },
-});
+  appendTo: "#search_form > div"
+}).data("ui-autocomplete")._renderMenu = function(ul, items) {
+  ul.addClass("typeahead dropdown-menu");
+  var that = this;
```

Comment:
First, this should be a `let` but second, why are we even creating `that` in the first place? Let's just use `this`